src/Resources/config/doctrine/Document.orm.xml

```diff
@@ -8,19 +8,16 @@
         <id name="id" type="string" column="id">
             <generator strategy="NONE"/>
         </id>
-        <id name="indexName" type="string" column="indexName">
-            <generator strategy="NONE"/>
-        </id>
 
         <field name="type" type="string" column="type" length="255" nullable="false"/>
         <field name="data" type="json_array" column="data" nullable="false"/>
 
         <one-to-many field="fields"
                      target-entity="Pucene\Bundle\DoctrineBundle\Entity\Field"
-                     mapped-by="document">
-            <cascade>
```

Comment:
the one to many should be marked as cascade-remove when the DB is not supporting native onCascade delete.